src/lighthouse.rs

```diff
@@ -90,55 +113,83 @@ fn quorum_changed(a: &Vec<QuorumMember>, b: &Vec<QuorumMember>) -> bool {
 }
 
 // Checks whether the quorum is valid and an explanation for the state.
-fn quorum_valid(state: &RoomState, opt: &LighthouseOpt) -> (bool, String) {
-    let mut first_joined = Instant::now();
+fn quorum_valid(
+    now: Instant,
+    heartbeats: &HashMap<String, Instant>,
+    state: &RoomState,
+    opt: &LighthouseOpt,
+) -> (bool, String) {
+    let mut first_joined = now;
```

Comment:
Not a request for change but a clarification. This actually isnt needed right? first joined will be min(join time of participants waiting in room).

Or is it used for if theres no participants in the room? I would think min replica should be the one catching this.

Reply:
that's basically what this is computing -- this is just the initial value before we calculate the minimum